src/vtt-segment-loader.js

```diff
@@ -976,6 +980,22 @@ export default class VTTSegmentLoader extends videojs.EventTarget {
 
     segment.requested = true;
 
+    // prepend the media initialization segment if it exists
+    if (segment.map) {
+      const initId = initSegmentId(segment.map);
+      const initSegment = this.initSegments_[initId];
+
+      segment.map.bytes = initSegment.bytes;
+
+      const vttHeader = new Uint8Array('WEBVTT\n\n'.split('').map(char => char.charCodeAt(0)));
+      const combinedByteLength = vttHeader.byteLength + segmentInfo.bytes.byteLength;
+      const combinedSegment = new Uint8Array(combinedByteLength);
+
+      combinedSegment.set(vttHeader);
+      combinedSegment.set(segmentInfo.bytes, vttHeader.byteLength);
+      segmentInfo.bytes = combinedSegment;
```

Comment:
Do we need combined if we later parse the init segment before the segment itself?